### PR TITLE
create ppc_dashboard

### PR DESCRIPTION
**Pull request by yilinxia**
#7

### CODE REVIEW

File: arviz_dashboard/ppc.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import arviz as az
+import bokeh.io
+import matplotlib.pyplot as plt
+import panel as pn
+from bokeh.palettes import Category10
+from bokeh.plotting import figure
+import time
```
Comment (aloctavodia) [inline arviz_dashboard/ppc.py:7]: unused import

File: arviz_dashboard/__init__.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 """Bayesian Dashboards built on top of Panel"""
 __version__ = "0.0.1a"
 
-from .elpd import dashboard_elpd
\ No newline at end of file
+from .elpd import dashboard_elpd
+from .ppc import dashboard_ppc
```
Comment (aloctavodia) [inline arviz_dashboard/__init__.py:5]: Not necessary to change now, but we should think of better names for the dashboard functions. If we are going to import `arviz_dashboard as azd` or something similar then we could omit the dashboard from the names.

**Comment by aloctavodia** [comment]
@yilinxia let me know if you need more feedback of help

File: arviz_dashboard/ppc.py (hunk 1 above)
Comment (yilinxia) [reply arviz_dashboard/ppc.py:7]: resolved

File: arviz_dashboard/__init__.py (hunk 2 above)
Comment (yilinxia) [reply arviz_dashboard/__init__.py:5]: Yeah,I do think we should think another name for this. dashboard_elpd sames too long

**Comment by yilinxia** [comment]
Hi @aloctavodia , I am so sorry that I totally forgot to push the changes. Please let me know if there are other problems

**Pull request merged**